HISTORY.md

```diff
@@ -1386,7 +1386,7 @@
 	- [Chai](http://chaijs.com/) 1.1.x to 1.3.x
 - Removed ability to require uncompiled plugins
 	- This is due to the CoffeeScript 1.4.x from 1.3.x upgrade
-- `skeletonNonexistant` now tells us the path it checked
+- `skeletonNonexistent` now tells us the path it checked
```

Comment:
'Existant' seems to be a French word. 
http://www.wordreference.com/fren/existant

'existant' is referred as an alternative spelling for 'existent' here.
https://en.wiktionary.org/wiki/existant

Don't know any more. Thanks